webdriver/tests/perform_actions/pointer_tripleclick.py

```diff
@@ -19,8 +19,8 @@ def test_tripleclick_at_coordinates(session, mouse_chain):
     div = session.find.css("div", all=False)
     div_rect = div.rect
     div_centre = {
-        "x": div_rect["x"] + div_rect["width"]/2,
-        "y": div_rect["y"] + div_rect["height"]/2
+        "x": div_rect["x"] + int(div_rect["width"]/2),
+        "y": div_rect["y"] + int(div_rect["height"]/2)
```

Comment:
I don't quite follow this change - does the logic need this to be integer division and so broke when this became a float?

Reply:
Yes. Otherwise it raises the following error:
```
E           webdriver.error.InvalidArgumentException: invalid argument (400): invalid argument
E           from invalid argument: 'x' must be an int
```
It's not an issue in Python 2 as an integer division returns an integer.

Reply:
Ok, sgtm.